cypress/e2e/audio-player/audio-player.spec.ts

```diff
@@ -48,6 +48,8 @@ describe("Audio player", () => {
     /**
      * Take snapshots of mx_EventTile_last on each layout, outputting log for reference/debugging.
      * @param detail The Percy snapshot name. Used for outputting logs too.
+     * @param monospace This enables a monospace font installed locally. Set to false by default.
+     * Please note that the specific font applied to Percy snapshots can be different among environments.
```

Comment:
```suggestion
     * @param monospace This causes a monospace font to be installed locally. Set to false by default.
     *     Note that the specific font applied to Percy snapshots can be different among environments.
```

Reply:
(does it really cause a font to be installed? or does it just enable a setting to use it?)

Reply:
oh, maybe I misunderstood you? Do you mean "This enables a monospace font, which must already be installed locally"? But then, what does it mean about installation?

Reply:
Its intention is to avoid confusion about difference of monospace fonts between the snapshots on Percy and what you would see when you run the test locally. It seems that one of the monospace fonts which has been installed and is available on the environment of Percy is picked and used when snapshots are taken (apparently there is not a font which is named as `monospace`).

As I compared snapshots and local Cypress result by myself, there was in fact such a difference on my environment.

Reply:
> I am wondering whether `This changes the font used to render the UI from a default one to a monospace one available locally. Set to false by default. Note that the font applied to Percy snapshots can be different from the the test result on your local environment` would be clearer.

yes, I think that is clearer, though I still think that "available locally" is confusing. Maybe just omit it?

Reply:
I would agree. I am going to change the commend with a commit.

Reply:
@richvdh Done :v: https://github.com/matrix-org/matrix-react-sdk/pull/10441/commits/e1ae92b85e7de3d2cd584bfbfa3ccf81a97042d9